src/mumble/GlobalShortcut.cpp

```diff
@@ -849,6 +849,9 @@ bool GlobalShortcutEngine::handleButton(const QVariant &button, bool down) {
 
 	if (down) {
 		AudioInputPtr ai = g.ai;
+		if (ai->activityState == AudioInput::ActivityStateIdle) {
+			ai->activityState = AudioInput::ActivityStateExitedIdle;
```

Comment:
Yeah, good point. I think a race problem is unlikely here do to the Active -> Idle -> ReturnedFromIdle cyclic nature of this state variable, but that may change in the future and bite us if it's unprotected.

Once this PR is through, I can submit a separate one that locks tIdle / activityState behind a QMutex.